Tools/wasm/mypy.ini

```diff
@@ -0,0 +1,15 @@
+[mypy]
+files = Tools/wasm
+pretty = True
+show_traceback = True
+
+# Make sure the wasm can be run using Python 3.8:
+python_version = 3.8
```

Comment:
I'm worried that having to support that far back is going to be more of a burden than benefit to keep this code typed since access to `typing_extensions` is probably not an option and this code is not evolving or exposed as an API.

Reply:
We can support 3.10+ as other tools do, but, I've seen this line: https://github.com/python/cpython/blob/b4131a13cb41d0f397776683c3b99500db9e2cfd/Tools/wasm/wasm_build.py#L42 And I've decided to stick with it, since no changes are basically required.

Reply:
Since I'm planning to write some new code in a separate script that will probably be untyped to start, we can stick with Python 3.8 in this file and if it becomes a burden we can raise the minimum Python version.